app/src/main/java/com/eveningoutpost/dexdrip/MegaStatus.java

```diff
@@ -439,21 +439,20 @@ protected void onResume() {
     @Override
     public boolean onCreateOptionsMenu(Menu menu) {
         // Inflate the menu; this adds items to the action bar if it is present.
-        //getMenuInflater().inflate(R.menu.menu_mega_status, menu);
+        getMenuInflater().inflate(R.menu.menu_mega_status, menu);
         return true;
     }
 
+    public void viewEventLog(MenuItem x) {
+        startActivity(new Intent(this, EventLogActivity.class).setFlags(Intent.FLAG_ACTIVITY_NEW_TASK).putExtra("events", ""));
+    }
+
     @Override
     public boolean onOptionsItemSelected(MenuItem item) {
         // Handle action bar item clicks here. The action bar will
         // automatically handle clicks on the Home/Up button, so long
         // as you specify a parent activity in AndroidManifest.xml.
-        int id = item.getItemId();
-
-        //noinspection SimplifiableIfStatement
-        if (id == R.id.action_settings) {
-            return true;
-        }
+        //int id = item.getItemId();
 
         return super.onOptionsItemSelected(item);
```

Comment:
If you just call the super method, you can remove this overriding function completely.

Reply:
I think it's called behind the scenes, instead of explicitly, so the way to do that would presumably be to delete the function entirely, or at least comment it out. Do you have a preference for one or the other?

Reply:
If a function does nothing else than its super function it is better to remove it. This removes code readability. In our case, the click is simply handled and we don't do anything special.

Reply:
I've removed the function. 👍